src/main/resources/templates/analysis_index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="UTF-8">
+    <title>JStat Analysis</title>
+</head>
+<body>
+
+<form method="POST" th:action="@{/analysis}" target="_blank">
+
+<ul>
+
+    <li>Select Analysis Type</li>
+    <ul>
+        <li>
+            <input name="eda" type="radio" value="exploratory_analysis" /> Exploratory Analysis
+        </li>
+        <li>
+            <input name="plotting" type="radio" value="plot" /> Plot
+        </li>
+        <li>Modelling
+            <ul>Regression
+                <li>
+                    <input name="linear_regression" type="radio" value="linear_regression" /> Linear Regression
+                </li>
+                <li>
+                    <input name="non_linear_regression" type="radio" value="non_linear_regression" /> Non Linear Regression
+                </li>
+            </ul>
+            <ul>Classification
+                <li>
+                    <input name="logistic_regression" type="radio" value="logistic_regression" /> Logistic Regression
+                </li>
+            </ul>
+            <ul>Clustering
+                <li>
+                    <input name="kmeans_clustering" type="radio" value="kmeans" /> KMeans
+                </li>
+            </ul>
+        </li>
+    </ul>
+
+
+    <li>Select active dataset</li>
+    <p>This should be a list of loaded datasets</p>
+
+    <div th:each="set : ${dataSets}">
+        <input name="dataSetName" type="radio" th:text="${set.name}"  th:value="${set.name}" required/>
+        <select th:id="${set.name}" name="colName">
+            <option th:each="value : ${set.names}" th:value="${value}" th:text="${value}" ></option>
+        </select>
+    </div>
+
+</ul>
+
+
+<div>
+
+</div>
+    <button>Do Analysis</button>
+</form>
+
+</body>
+</html>
```

Comment:
we can use the frontend files from the redefine_beautify branch, I think. 
but they are ok for now.

Reply:
ok